src/module/Normalizer/src/Normalizer/View/Helper/Normalize.php

```diff
@@ -53,33 +56,101 @@ public function headMeta($object)
         $robots     = $metadata->getRobots();
         $preview    = $this->toPreview($object);
         $image      = $this->getMetaImage($object);
-        $meta->setProperty('og:title', $title);
-        $meta->setProperty('og:image', $image);
+        $site_name  = $this->getView()->brand()->getBrand(true);
+
         $meta->appendName('content_type', $type);
         $meta->appendName('description', $preview);
         $meta->appendName('keywords', implode(', ', $keywords));
         $meta->appendName('robots', $robots);
+        $meta->setProperty('og:title', $title);
+        $meta->setProperty('og:type', 'website');
+        $meta->setProperty('og:image', $image);
+        $meta->setProperty('og:description', $preview);
+        $meta->setProperty('og:site_name', $site_name);
+        $meta->setProperty('fb:pages', '155020041197918');
+        $meta->setProperty('fb:profile_id', '155020041197918');
+        $this->getView()->headLink(['rel' => 'search', 'href' => '/opensearch.xml', 'title' => 'Serlo (de)','type' => 'application/opensearchdescription+xml']);
 
         return $this;
     }
 
+
     public function getMetaImage($object)
     {
-        //TODO: Change path depending ob subject
-        return 'https://de.serlo.org/assets/images/meta_serlo.jpg';
+        $fileName = 'meta_serlo.jpg';
+
+        $subject = trim(strtolower(strip_tags(
+            $this->getView()->navigation('default_navigation')
+                ->menu()
+                ->setPartial('layout/navigation/partial/active-subject')
+                ->setOnlyActiveBranch(true)
+                ->setMinDepth(0)
+                ->setMaxDepth(0)
+                ->render()
+        )));
+
+        switch ($subject) {
+            case 'mathematik':
+            case 'math':
```

Comment:
IMHO: either we add English versions of the images or remove the behaviour for `en.serlo.org` for now.

Reply:
@Knorrke: I removed the handling for subjects on en.serlo.org for now (i.e. the fallback image is used instead). Are you fine with that?

Reply:
Yeah, that's fine I guess